src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -74,9 +74,9 @@ protected Stream GetNodeItemDataStream(IEnumerable<NodeSearchElement> searchEntr
         protected virtual object CreateObjectForSerialization(IEnumerable<NodeSearchElement> searchEntries)
         {
             var data = new LoadedTypeData<LoadedTypeItem>();
-            data.loadedTypes = searchEntries
-                //.Where(e => !e.ElementType.HasFlag(ElementTypes.Packaged))
```

Comment:
@aparajit-pratap @reddyashish @ColinDayOrg - it just occurs to me that this commented out code is kind of peculiar, does anyone know why this was left in?

Reply:
Sorry was waiting for others to see if they knew about this. I am not sure why that line was commented in the first place. I assumed we are not using that attribute filter anymore.  

The changes look good.